### PR TITLE
Update PercentComplete to display 1/10th of the Progress for reindexing

### PR DESCRIPTION
**Pull request by apurvabhaleMS**
## Description
Updated PercentComplete  to display 1/10th of the progress when query for the reindexing status
Do not output fields with no values

## Related issues
Addresses [#AB81444](https://microsofthealth.visualstudio.com/Health/_boards/board/t/Resolute/Stories/?workitem=81444)

## Testing
Unit test
Manual Testing

## FHIR Team Checklist
- [x] **Update the title** of the PR to be succinct and less than 50 characters
- [x] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- [x] Tag the PR with the type of update: **Bug**, **Dependencies**, **Enhancement**, or **New-Feature**
- [x] Tag the PR with **Azure API for FHIR** if this will release to the managed service
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking (reason)

### CODE REVIEW

File: src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/Models/ReindexJobRecord.cs [hunk 1]
```diff
@@ -115,13 +115,13 @@ protected ReindexJobRecord()
...
         [JsonIgnore]
-        public int PercentComplete
+        public decimal PercentComplete
         {
             get
             {
                 if (Count > 0 && Progress > 0)
                 {
-                    return (int)((double)Progress / Count * 100);
+                    return (decimal)Math.Round((double)Progress / Count * 100, 1);
```
Comment (Ivanidzo4ka) [inline src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/Models/ReindexJobRecord.cs:124]: >decimal [](http://example.com/codeflow?start=28&length=7)

super nit:
why cast it to `decimal` if it's already `double`?

Especially if we display only one digit ;)

File: src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/Models/ReindexJobRecord.cs [hunk 2]
```diff
@@ -115,13 +115,13 @@ protected ReindexJobRecord()
         public IReadOnlyCollection<string> TargetResourceTypes { get; private set; } = new List<string>();
 
         [JsonIgnore]
-        public int PercentComplete
+        public decimal PercentComplete
```
Comment (Ivanidzo4ka) [inline src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/Models/ReindexJobRecord.cs:118]: >PercentComplete [](http://example.com/codeflow?start=23&length=15)

another nit:
Can we actually move it to `ToParametersResourceElement` function?
It's only used there, so I don't see much value in having this property here.

Comment (apurvabhaleMS) [reply src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/Models/ReindexJobRecord.cs]: Done

**Review by Ivanidzo4ka — APPROVED** [review]
:shipit:

**Pull request merged**